src/main/java/com/spring/onlinebookstore/model/CartItem.java

```diff
@@ -0,0 +1,35 @@
+package com.spring.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+
+@Entity
+@Table(name = "cart_items")
+@Getter
+@Setter
+public class CartItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "shopping_cart_id")
+    @Fetch(FetchMode.JOIN)
+    private ShoppingCart shoppingCart;
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "book_id", nullable = false)
+    @Fetch(FetchMode.JOIN)
+    private Book book;
```

Comment:
Check if you need it

Reply:
Looks like you don`t, because I don`t see logic when this is used